src/stan/services/util/create_rng.hpp

```diff
@@ -19,14 +19,17 @@ namespace util {
  * duplicated.
  *
  * @param[in] seed the random seed
- * @param[in] chain the chain id
+ * @param[in] init_chain_id the chain id
+ * @param[in] chain_num For multi-chain, the ch
```

Comment:
doc got cut?

Reply:
Nope! Names of the arguments changed

Reply:
But the doc "For multi-chain, the ch" is not meaningful to me. What about 

```
@param[in] init_chain_id start of chain ids
 @param[in] chain_num chain id offset such that chain_id is init_chain_id+chain_num
  ```

Also... reading the comment here suggest to me that we should actually not allow for a chain id of 0, right??? (I mean the comment to the function)

Reply:
BTW, do we actually need to have `create_rng` take a `init_chain_id` and a `num_chains`? Why don't we leave the function as is and just pre-compute the chain_id's in the parallel sample functions? That seems simpler to me.

Reply:
No we do not need to change this good find! I'll switch it back